src/olympia/constants/base.py

```diff
@@ -443,3 +443,30 @@
 
 DISCO_API_ALLOWED_PARAMETERS = (
     'telemetry-client-id', 'lang', 'platform', 'branch', 'study', 'edition')
+
+DOWNLOAD_SOURCES_FULL = (
```

Comment:
Maybe add a comment linking to all our officially allowed download sources (the one we describe in the api docs) and make a note that the rest is here potentially for historic reasons.